ts/features/mvl/types/mvlData.ts

```diff
@@ -0,0 +1,61 @@
+import { IUnitTag } from "@pagopa/ts-commons/lib/units";
+import { ValidUrl } from "@pagopa/ts-commons/lib/url";
+import { ContentType } from "../../../types/contentType";
+import { Byte } from "../../../types/digitalInformationUnit";
+
+/**
+ * The unique ID of a MVLId, used to avoid to pass wrong id as parameters
+ */
+export type MvlId = string & IUnitTag<"MvlId">;
+
+export type WithMVLId<T> = T & {
+  id: MvlId;
+};
+
+/**
+ * The content of the MVL with two possible representation
+ */
+export type MvlBody = {
+  html: string;
+  plain?: string;
+};
+
+/**
+ * Represent an attachment with the metadata and resourceUrl to retrieve the attachment
+ */
+export type MvlAttachment = {
+  // a display name for the file
+  name: string;
+  // atm we have to distinguish only the pdf files from the others for a custom (future) view
+  contentType: Extract<ContentType, "application/pdf"> | "other";
+  // size (in Byte) of the attachment, for display purpose
+  size: Byte;
+  // The url that can be used to retrieve the resource
+  resourceUrl: ValidUrl;
+};
+
+/**
+ * Additional metadata
+ * TODO: Just an initial stub, should be completed and refined
+ */
+export type MvlMetadata = {
+  sender: string;
+  receiver: string;
+  cc: ReadonlyArray<string>;
```

Comment:
```suggestion
  sender: string;
  receiver: string;
  cc: ReadonlyArray<string>;
```
Could it be an improvement using the existing type `EmailAddress`?

Reply:
Better for sure, the more types the merrier!
Changed in a8093521bcab81608fea29e2d159eb8087aa5f17